cmd/encryption-v1.go

```diff
@@ -99,15 +80,15 @@ const (
 //
 //   Seal:
```

Comment:
You can also remove the entire comment section here - `crypto/doc.go` contains doc.